### PR TITLE
[Compute] Powershell changes for RunCommand, VMSS Scale In Policy, RestorePointsCollection

### PR DESCRIPTION
**Pull request by haagha**
## Description

(https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1152)
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1072

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by haagha** [comment]
@isra-fel please review these changes for upcoming release.

File: src/Compute/Compute/help/Update-AzVmss.md [hunk 1]
```diff
@@ -723,6 +723,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ScaleInPolicyForceDeletion
+{{ Fill ScaleInPolicyForceDeletion Description }}
```
Comment (isra-fel) [inline src/Compute/Compute/help/Update-AzVmss.md:727]: Fill ScaleInPolicyForceDeletion Description

Comment (isra-fel) [inline src/Compute/Compute/help/Update-AzVmss.md:727]: Fill ScaleInPolicyForceDeletion Description

**Review by isra-fel — CHANGES_REQUESTED** [review]
Code complete has passed. This PR will go into the May release.

**Review by wyunchi-ms — APPROVED** [review]

**Comment by wyunchi-ms** [comment]
Ping @haagha

**Pull request closed without merging**